### PR TITLE
CID-2533 Update federation documentation

### PR DESCRIPTION
**Pull request by nikhil09541**
- Customers are right now confused on admin role which provides them full access to all cloud products. Customers think that this includes billing and ticketing and would like to call out that this doesn’t not include Billing and Ticketing.
-Changed the docs for the above point.

### CODE REVIEW

**Comment by rackernexus** [comment]
Build 132 was successful. Staging content [Preview!](https://developer-revs.staging.deconst.org/build-194644932/docs/rackspace-federation/). valid for 14 days

File: doc/config-attribute-mapping/full-roles.rst [hunk 1]
```diff
@@ -16,10 +16,10 @@ and role for the capabilities that you want to provide.
      - Description
      - Role name
    * - Global RBAC roles (applies to all products)
-     - Administrator (full control) for any Cloud product
+     - Grants administrator (full control) for any Cloud products, this does not include access to Billing or Ticketing
```
Comment (timcline) [inline doc/config-attribute-mapping/full-roles.rst:19]: Can you reference the roles required for that here?

**Comment by WilliamLoy** [comment]
@catlook did a full audit on this one but got a little crosseyed toward the end. In any case I believe this is ready for your review.

**Comment by rackernexus** [comment]
Build 133 was successful. Staging content [Preview!](https://developer-revs.staging.deconst.org/build-fb4f7b46b/docs/rackspace-federation/). valid for 14 days

**Comment by catlook** [comment]
@WilliamLoy On it!

**Comment by rackernexus** [comment]
Build 134 was successful. Staging content [Preview!](https://developer-revs.staging.deconst.org/build-71d9cebd7/docs/rackspace-federation/). valid for 14 days

**Comment by rackernexus** [comment]
Build 135 was successful. Staging content [Preview!](https://developer-revs.staging.deconst.org/build-3ee639f8a/docs/rackspace-federation/). valid for 14 days

**Comment by rackernexus** [comment]
Build 136 was successful. Staging content [Preview!](https://developer-revs.staging.deconst.org/build-d0eff007e/docs/rackspace-federation/). valid for 14 days

**Comment by catlook** [comment]
@WilliamLoy I have completed my edits.  Review them and see if you agree!

**Pull request merged**